contracts/SupplyChain.sol

```diff
@@ -0,0 +1,25 @@
+pragma solidity 0.5.0;
+
+import "./Package.sol";
+
+
+contract SupplyChain {
+
+    mapping(bytes => Package) private packages;
+
+    function getPackage(bytes memory _packageId) public view returns (Package) {
```

Comment:
Does it return struct correctly?

Reply:
`Package` is a contract so it returns the contract's address.